src/App.jsx

```diff
@@ -14,70 +17,129 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const [isSelected, setIsSelected] = useState(true);
+
+  const checkSelectingGood = (row) => {
+    if (isSelected) {
+      const selected = document.querySelector(
+        '.has-background-success-light',
+      );
+
+      if (selected !== row) {
+        const button = selected.querySelector('.button');
+
+        selected.classList.remove(
+          'has-background-success-light',
+        );
+        button.innerText = '+';
+        button.dataset.cy = 'AddButton';
+        button.classList.remove('is-info');
```

Comment:
Don't interact with DOM directly. You can just change the class of the good according to the state as follow
`<div class={true ? 'active' : ''}>`